--- a/meson.build	2025-07-13 08:19:44.922358518 -0600
+++ b/meson.build	2025-07-13 08:20:41.535941691 -0600
@@ -81,13 +81,12 @@
 RUNSTATEDIR_INSTALLED = PREFIX / RUNSTATEDIR
 SHAREDSTATEDIR = LOCALSTATEDIR / 'lib/kea'
 SHAREDSTATEDIR_INSTALLED = PREFIX / SHAREDSTATEDIR
-SYSCONFDIR_INSTALLED = PREFIX / SYSCONFDIR
+SYSCONFDIR_INSTALLED = SYSCONFDIR
 TEST_CA_DIR = TOP_SOURCE_DIR / 'src/lib/asiolink/testutils/ca'
 
 # Meson is annoying with its opinionated alteration of certain paths based on whether prefix is default or not.
 # So we revert what it does.
 if PREFIX == '/usr/local'
-    SYSCONFDIR_INSTALLED = '/usr/local/etc'
     LOGDIR_INSTALLED = '/usr/local/var/log/kea'
     RUNSTATEDIR_INSTALLED = '/usr/local/var/run/kea'
     SHAREDSTATEDIR_INSTALLED = '/usr/local/var/lib/kea'
