diff --git a/wiretap/erf.c b/wiretap/erf.c
index 785448ad3c..a4b6403683 100644
--- a/wiretap/erf.c
+++ b/wiretap/erf.c
@@ -3224,8 +3224,8 @@ static int populate_summary_info(erf_t *erf_priv, wtap *wth, wtap_rec *rec, uint
 {
   struct erf_meta_read_state state = {0};
   struct erf_meta_read_state *state_post = NULL;
-  uint64_t host_id;
-  uint8_t source_id;
+  uint64_t host_id = 0;
+  uint8_t source_id = 0;
   GList *post_list = NULL;
   GList *item = NULL;
 
